src/controllers/image-request.ts

```diff
@@ -67,37 +67,17 @@ interface HandlerResponse {
 export const handle = async (
   request: CloudFrontRequest,
 ): Promise<HandlerResponse> => {
-  const uri = request.uri;
-
-  if (uri === undefined) {
-    return {
-      statusCode: '400',
-      body: 'Invalid Request',
-      headers: {
-        'Content-Type': 'text/plain',
-      },
-    };
-  }
-
-  logger.info({
-    message: 'valid uri',
-    uri,
-  });
-
   const queryString = new URLSearchParams(request.querystring);
 
-  logger.info({
-    queryString,
-  });
-
+  const getFromExternal = queryString.get('getFromExternal') === 'true';
```

Comment:
Few things.

We should not allow this from request parameters. This should instead be set as an environment variable with the base domain name passed in as this allows any end user to use our service as a way to hit any site. Not great!